src/frontends/ir/src/ir_deserializer.cpp

```diff
@@ -596,10 +596,19 @@ class MetaDataParser : public ov::Meta {
     }
 
     operator ov::AnyMap&() override {
+        m_unchanged_flag = false;
         parse();
         return m_parsed_map;
     }
 
+    pugi::xml_node get_pugi_node() const override {
```

Comment:
if we change it to raw ptr, we could return NULL for case when meta data has changed.

Reply:
pugi::xml_node is a custom view_pointer so we can compare it with NULL